lib/linguist/languages.yml

```diff
@@ -404,6 +404,16 @@ Arc:
   tm_scope: none
   ace_mode: text
   language_id: 20
+ArkTS:
+  type: programming
+  color: "#3178c6"
+  extensions:
+  - ".ets"
+  tm_scope: source.ets
+  ace_mode: typescript
+  codemirror_mode: javascript
+  codemirror_mime_type: application/typescript
+  language_id: 1000001
```

Comment:
Please follow the instructions in the CONTRIBUTING.md file for generating this ID.